release-notes/2.1/releases.json

```diff
@@ -9622,7 +9622,7 @@
             "hash": "405cbd7c65d63b36e3bd6bcdfc897ac6474c4eaf93db9db478a80ab511bfa7a1c4a84024cc6e4af0df0af86bcc0a1a96a8ba0864c77bf579f32bce437c28d5a8"
```

Comment:
Can we add the missing Rid property for x86 too?

Reply:
good catch - done